tests/functional/src/main/java/org/mule/functional/classloading/isolation/api/ArtifactClassLoaderHolder.java

```diff
@@ -5,25 +5,25 @@
  * LICENSE.txt file.
  */
 
-package org.mule.functional.classloading.isolation.classification;
+package org.mule.functional.classloading.isolation.api;
```

Comment:
I think that the api package must be shared across all the things that we consider API on the functional tests, i.e., org.mule.functional.api.classloading.isolation